src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs

```diff
@@ -29,7 +29,9 @@ public enum DatabaseItemType
         CompareExchangeTombstones = 1 << 17,
         TimeSeries = 1 << 18,
 
-        ReplicationHubCertificates = 1 << 19
+        ReplicationHubCertificates = 1 << 19,
+
+        TimeSeriesDeletedRanges = 1 << 20
```

Comment:
Do we need to handle this in 'ApplyBackwardCompatibility' method in the smuggler handler?

Reply:
not sure..
do we want older clients to operate on `TimeSeriesDeletedRanges` items? 
always? 
or only if operating on `TimeSeries` items?

Reply:
Any more thoughts on this? CC @karmeli87

Reply:
done
for older clients  (5.0 <= version < 5.4.200) that operate on `TimeSeries` items, we add `TimeSeriesDeletedRanges`  to `OperateOnTypes`